src/app/home/home.component.html

```diff
@@ -283,11 +283,9 @@
                         }"
                     >
                         <div class="grid1">
-                            <span
-                                class="text-marking {{ textMarking.type }}"
-                                data-test="text-marking-span"
```

Comment:
Why has `data-test="text-marking-span"` been removed? Additionally, why have `cursor-pointer` removed along with `/` in the routerLink? Perhaps main didn't get all latest changes.

Reply:
> Why has `data-test="text-marking-span"` been removed? Additionally, why have `cursor-pointer` removed along with `/` in the routerLink? Perhaps main didn't get all latest changes.

It must have been a merge accident by me that's why it has been removed